.fides/db_dataset.yml

```diff
@@ -1474,15 +1474,6 @@ dataset:
     - name: created_at
       data_categories: [system.operations]
       data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
-    - name: delivery_mechanism
-      data_categories: [system.operations]
-      data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
```

Comment:
Ah this should have been removed earlier, note to self that the fides scan doesn't detect extra fields